pkg/workloads/lib/package.py

```diff
@@ -50,17 +65,31 @@ def build_packages(python_packages, bucket):
 
     logger.info("Setting up packages")
 
+    restricted_packages = get_restricted_packages()
+
     for package_name in build_order:
+        package_wheel_path = os.path.join(WHEELHOUSE_PATH, package_name)
         requirement = cmd_partial[package_name]
         logger.info("Building package {}".format(package_name))
         completed_process = run(
-            "pip3 wheel -w {} {}".format(
-                os.path.join(WHEELHOUSE_PATH, package_name), requirement
-            ).split()
+            "pip3 wheel -w {} {}".format(package_wheel_path, requirement).split()
         )
         if completed_process.returncode != 0:
             raise UserException("creating wheels", package_name)
 
+        for wheelname in os.listdir(package_wheel_path):
+            name_split = wheelname.split("-")
+            dist_name = name_split[0]
```

Comment:
nit: this might be a little cleaner if it's `dist_name, version = name_split[0], name_split[1]`, or if there is only one `-`, you can do `dist_name, version = wheelname.split("-")`